File: Corte 1/Retos/reto2.cpp
#include <iostream>
#include <iomanip>
using namespace std;

/*
 * RETO 2
 * El centro de salud Famisalud aplica vacunas a bebes menores de un año.
 * La dosis depende del peso y la edad del bebe segun la formula:
 *
 *      dosisvacuna = ((pesobebe + 10) / (mesesbebe * 10)) * 8
 */

int main() {
    double pesoBebe, mesesBebe;

    cout << "=====================================" << endl;
    cout << "   RETO 2 - Centro de Salud Famisalud" << endl;
    cout << "=====================================" << endl;

    cout << "Ingrese el peso del bebe (en kg): ";
    cin >> pesoBebe;

    cout << "Ingrese la edad del bebe (en meses): ";
    cin >> mesesBebe;

    // Aplicamos la formula: dosisvacuna = ((pesobebe + 10) / (mesesbebe * 10)) * 8
    double dosisVacuna = ((pesoBebe + 10) / (mesesBebe * 10)) * 8;

    cout << "-------------------------------------" << endl;
    cout << fixed << setprecision(2);
    cout << "La dosis de vacuna a aplicar es: " << dosisVacuna << " ml" << endl;
    cout << "=====================================" << endl;

    return 0;
}

File: Corte 1/Retos/reto2.py
"""
RETO 2
El centro de salud Famisalud aplica vacunas a bebes menores de un año.
La dosis depende del peso y la edad del bebe segun la formula:

    dosisvacuna = ((pesobebe + 10) / (mesesbebe * 10)) * 8
"""


def main():
    print("   RETO 2 - Centro de Salud Famisalud")
    
    peso_bebe = float(input("Ingrese el peso del bebe (en kg): "))
    meses_bebe = float(input("Ingrese la edad del bebe (en meses): "))

    # Aplicamos la formula: dosisvacuna = ((pesobebe + 10) / (mesesbebe * 10)) * 8
    dosis_vacuna = ((peso_bebe + 10) / (meses_bebe * 10)) * 8

    print(f"La dosis de vacuna a aplicar es: {dosis_vacuna:.2f} ml")
    
if __name__ == "__main__":
    main()